packages/packages/documents/src/sync/utils.ts

```diff
@@ -11,11 +11,17 @@ export function getV1DocumentsManager() {
 }
 
 export function normalizeV1Document( documentData: V1Document ): Document {
+	// TODO: See: https://github.com/elementor/elementor/blob/e86b957a2/assets/dev/js/editor/document/save/behaviors/footer-saver.js#L96-L101
+	const status = documentData.container.settings.get( 'post_status' ) ?? 'pending';
+
+	const isAutoSave = documentData.config.revisions.current_id !== documentData.id;
```

Comment:
```suggestion
	const isUnpublishedRevision = documentData.config.revisions.current_id !== documentData.id;
```

with comment of draft and autosave

Reply:
Done